src/games/strategy/triplea/ai/proAI/ProData.java

```diff
@@ -0,0 +1,43 @@
+package games.strategy.triplea.ai.proAI;
+
+import games.strategy.engine.data.GameData;
+import games.strategy.triplea.Properties;
+
+/**
+ * Pro AI data.
+ */
+public class ProData {
+
+  public static double winPercentage = 95;
+  public static double minWinPercentage = 75;
+  public static boolean areNeutralsPassableByAir = false;
+
+  private static GameData data;
+  private static ProAI proAI;
+
+  public static GameData getData() {
+    return data;
+  }
+
+  public static void setData(final GameData data) {
+    ProData.data = data;
+
+    // Set optimal and win percentages lower if not LL
+    if (!games.strategy.triplea.Properties.getLow_Luck(data)) {
+      winPercentage = 90;
+      minWinPercentage = 65;
+    }
+
+    // Set map properties
+    areNeutralsPassableByAir = (Properties.getNeutralFlyoverAllowed(data) && !Properties.getNeutralsImpassable(data));
```

Comment:
Feature envy code smell. This class is envious of the data stored by Properties, namely the 'can fly over' and 'is impossible' properties. This boolean computation should probably live with the `Properties` class, ie: `Properties.getNeutralsPassableByAir(data)`